src/main/java/seedu/eduke8/command/BookmarkCommand.java

```diff
@@ -13,43 +14,49 @@
     private static final String BOOKMARK_STORE = "storing";
     private static final String BOOKMARK_DELETE = "delete";
 
-    private BookmarkList bookmarks;
+    private BookmarkList bookmarkList;
     private Question question;
     private boolean alreadyBookmarked = false;
     private int deleteIndex = 0;
     String typeOfBookmarkCommand = "";
 
-    public BookmarkCommand(Question question, String typeOfBookmarkCommand, BookmarkList bookmarks) {
+    public BookmarkCommand(Question question, String typeOfBookmarkCommand, BookmarkList bookmarkList) {
         super();
-        assert bookmarks != null;
+        assert bookmarkList != null;
         this.question = question;
         if (question.isBookmarked()) {
             alreadyBookmarked = true;
         } else {
-            bookmarks.add(question);
+            bookmarkList.add(question);
         }
-        this.bookmarks = bookmarks;
+        this.bookmarkList = bookmarkList;
         this.typeOfBookmarkCommand = typeOfBookmarkCommand;
     }
 
-    public BookmarkCommand(String typeOfBookmarkCommand, BookmarkList bookmarks) {
+    public BookmarkCommand(String typeOfBookmarkCommand, BookmarkList bookmarkList) throws Eduke8Exception {
         this.typeOfBookmarkCommand = typeOfBookmarkCommand;
-        this.bookmarks = bookmarks;
+        this.bookmarkList = bookmarkList;
+        if (bookmarkList.getCount() == 0) {
+            throw new Eduke8Exception("You currently do not have any bookmarks stored.");
+        }
     }
 
-    public BookmarkCommand(int deleteIndex, String typeOfBookmarkCommand, BookmarkList bookmarks) {
+    public BookmarkCommand(int deleteIndex, String typeOfBookmarkCommand, BookmarkList bookmarkList)
+            throws Eduke8Exception {
+        if (bookmarkList.getCount() == 0) {
+            throw new Eduke8Exception("You currently do not have any bookmarks stored.");
```

Comment:
Just need to abstract this magic string and should be good

Reply:
Remember to add the junits for this as well

Reply:
Good catch, will do so right away.